tools/run_tests/xds_k8s_test_driver/kubernetes-manifests/gamma/frontend_service.yaml

```
@@ -0,0 +1,10 @@
+---
+apiVersion: v1
+kind: Service          
+metadata:           
+  name: ${service_name}
+  namespace: ${namespace_name}
+spec:                   
+  ports:      
+  - port: 8080
+    targetPort: 8080
```

We should be passing `${test_port}` here
```suggestion
  - port: ${test_port}
    targetPort: ${test_port}
```

---

Next PR.